test/signalrclient-e2e-tests/connection_tests.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "stdafx.h"
+#include <string>
+#include "cpprest\details\basic_types.h"
+#include "cpprest\json.h" 
+#include "connection.h"
+#include "hub_connection.h"
+#include "stop_watch.h"
+
+extern utility::string_t url;
+
+TEST(connection_tests, connection_status_start_stop)
+{
+    stop_watch watch;
```

Comment:
I don't see how this makes the tests better (but I do see the opposite). Please remove this class and focus on addressing existing comments without gold-plating (http://en.wikipedia.org/wiki/Gold_plating_%28software_engineering%29)